csharp/ql/test/query-tests/Nullness/E.cs

```diff
@@ -366,6 +366,16 @@ static void Ex33(string s, object o)
     static int Ex34(string s = null) => s.Length; // BAD (maybe)
 
     static int Ex35(string s = "null") => s.Length; // GOOD
+
+    static int Ex44(object o)
```

Comment:
Why not `Ex36`?

Reply:
Now that is a very good question :-) I will change it.